pages/en/hosted-service/migrating-subgraph.mdx

```diff
@@ -4,11 +4,19 @@ title: Migrating an Existing Subgraph to The Graph Network
 
 ## Introduction
 
-This is a guide for the migration of subgraphs from the Hosted Service (also known as the Hosted Service) to The Graph Network. The migration to The Graph Network has been successful for projects like Opyn, UMA, mStable, Audius, PoolTogether, Livepeer, RAI, Enzyme, DODO, Opyn, Pickle, and BadgerDAO all of which are relying on data served by Indexers on the network. There are now over 200 subgraphs live on The Graph Network, generating query fees and actively indexing web3 data.
+This is a guide on how to migrate your subgraph from the Hosted Service to The Graph Network. The migration to The Graph Network has been successful for projects like Opyn, UMA, mStable, Audius, PoolTogether, Livepeer, RAI, Enzyme, DODO, Opyn, Pickle, and BadgerDAO all of which are relying on data served by Indexers on the network. There are now over 200 subgraphs live on The Graph Network, generating query fees and actively indexing web3 data.
 
-This will tell you everything you need to know about how to migrate to the decentralized network and manage your subgraphs moving forward. The process is quick and your subgraphs will forever benefit from the reliability and performance that you can only get on The Graph Network.
+The process of migration is quick and your subgraphs will forever benefit from the reliability and performance that you can only get on The Graph Network.
 
-### Migrating An Existing Subgraph to The Graph Network
+### When not to Migrate?
+
+If your subgraph is:
+
+- Indexing [IPFS](https://ipfs.io/).
+- Using [full-text search fields](https://thegraph.com/docs/en/developer/create-subgraph-hosted/#defining-fulltext-search-fields).
+- Deployed on other chains than Ethereum mainnet.
```

Comment:
"Indexing chains other than Ethereum mainnet" might be slightly clearer here, just to be consistent across the documentation (subgraphs are deployed to an instance of Graph Node).